### PR TITLE
Added implementation details for AbsoluteTelnet

### PR DESCRIPTION
**Pull request by rhyven**
Created absolutetelnet.yml in _data/impls/.  added protocol details.

(first pull request, hope I've done things right!)

### CODE REVIEW

File: _data/impls/absolutetelnet.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+name: AbsoluteTelnet
+homepage: www.celestialsoftware.com
```
Comment (fingolfin) [inline _data/impls/absolutetelnet.yml]: This should be a full URL, including protocol, i.e. add "http://". Also, I can't find anything about AbsoluteTelnet on that page. After searching, it seems you may have meant http://www.celestialsoftware.net/ ?

**Comment by fingolfin** [comment]
Thanks! Could you perhaps address the URL issue, then rebase your branch? I'll then merge it ASAP.

**Comment by rhyven** [comment]
@fingolfin -- fixed!  Thanks for the heads up  :)

**Comment by fingolfin** [comment]
I took the liberty of squashing this into a single commit, and added it. Thanks!

**Pull request closed without merging**